### PR TITLE
Update Binary.md

### PR DESCRIPTION
**Pull request by abhinavsri360**
I have added a small tutorial on how binary numbers are converted to Decimal numbers. This will help reader in getting the core idea of how numbers are converted.
The reason behind adding a tutorial is that there was no as such present. As the page describing what a binary number and also had a simulator. The only thing left was this little tutorial which will be a great asset for the learner.

---

## ✅️ By submitting this PR, I have verified the following

* [x] Checked to see if a similar PR has already been opened 🤔️
* [x] Reviewed the contributing guidelines 🔍️

### CODE REVIEW

**Comment by abhinavsri360** [comment]
Changes are reflected in the last commit i.e., f2df75b. All other commits do not bring any change to the repository.

**Comment by abhinavsri360** [comment]
I think that goes with the green theme of CircuitVerse and looks good which might tempt a reader to read. In case we use normal text might make it little dazy and anyone might give that a disinterested read. This is my opinion, please let me know if you still want code console to be removed.

**Comment by YashKumarVerma** [comment]
> I think that goes with the green theme of CircuitVerse and looks good which might tempt a reader to read. In case we use normal text might make it little dazy and anyone might give that a disinterested read. This is my opinion, please let me know if you still want code console to be removed.

I think we must follow the conventions as done on other pages. theory/text content is written as is, not as code.

**Comment by abhinavsri360** [comment]
Okay. I will surely make the changes and pull them in awhile.

**Comment by abhinavsri360** [comment]
image
Looks more clean now.

File: docs/binary.md [hunk 1]
```diff
@@ -81,31 +85,29 @@ It is just like counting in decimal except we reach 10 much sooner.
...
-<h3>See how it is done in this little demonstration</h3>
-
-```yaml
+## See how it is done in this little demonstration
+---
 Let's tell you something more about conversion. Conversion from Decimal to Binary is quite a simple task.
-All you need to do is begin from the right.
-STEP 1::
+All you need to do is begin from the right. Follow the steps below:<br>
+- STEP 1 :: <br>
```
Comment (YashKumarVerma) [inline docs/binary.md:92]: no hard br tag needed in markdown

File: docs/binary.md [hunk 2]
```diff
@@ -81,31 +85,29 @@ It is just like counting in decimal except we reach 10 much sooner.
...
+Result is ready :) <br>
+- NOTE :: <br>
 If the number is large, increase bits of binary number on the left. Keep in mind that it's value will
-increase subsiquently.
-Example ::
-Decimal Number :: 25
+increase subsiquently. <br>
+
+Example :: <br>
+Decimal Number :: 25 <br>
```
Comment (YashKumarVerma) [inline docs/binary.md:106]: no hard br tag needed in markdown

File: docs/binary.md [hunk 3]
```diff
@@ -81,31 +85,29 @@ It is just like counting in decimal except we reach 10 much sooner.
...
-Result is ready :)
-NOTE ::
+2^(place value from right) beginning from 0 i.e., 2^0, 2^1, 2^2 .... continuing upto 2^7. <br>
+- STEP 2 :: <br>
+Now, multiply each digit of binary number with it's value. <br>
+- STEP 3 :: <br>
+Add 'em all. <br>
+- STEP 4 :: <br>
+Result is ready :) <br>
+- NOTE :: <br>
```
Comment (YashKumarVerma) [inline docs/binary.md:101]: no hard br tag needed in markdown

File: docs/binary.md [hunk 4]
```diff
@@ -8,22 +8,26 @@ nav_order: 2
...
-Decimal number :: 25
-Binary Number :: 11001
 
-Note ::
+Example: <br>
+
+Decimal number :: 25 <br>
+Binary Number :: 11001 <br>
+
+Note :: <br>
```
Comment (YashKumarVerma) [inline docs/binary.md:27]: no hard br tag needed in markdown

**Comment by abhinavsri360** [comment]
waiting for a review @YashKumarVerma

**Pull request merged**